libc-top-half/musl/src/thread/wasm32/wasi_thread_start.s

```diff
@@ -0,0 +1,26 @@
+	.text
+
+	.export_name	wasi_thread_start, wasi_thread_start
+
+	.globaltype	__stack_pointer, i32
+	.functype	__wasi_thread_start_C (i32, i32) -> ()
+
+	.hidden	wasi_thread_start
```

Comment:
Logically (since this is an exported function) it maybe shouldn't be hidden?

Reply:
as far as i know, symbol visibility here is a completely separate concept from wasm export.